src/mpid/ch4/netmod/ofi/ofi_finalize.c

```diff
@@ -7,15 +7,89 @@
 #include "ofi_impl.h"
 #include "ofi_noinline.h"
 
+#define MPIDI_OFI_FLUSH_CONTEXT_ID 0xF000
+#define MPIDI_OFI_FLUSH_TAG        1
+
+/* send a dummy message to flush the send queue */
+static int flush_send(int dst)
+{
+    int mpi_errno = MPI_SUCCESS;
+
+    MPIR_Comm *comm = MPIR_Process.comm_world;
+    int data = 0;
+    uint64_t match_bits = MPIDI_OFI_init_sendtag(MPIDI_OFI_FLUSH_CONTEXT_ID,
+                                                 MPIDI_OFI_FLUSH_TAG, MPIDI_OFI_DYNPROC_SEND);
+
+    /* Use the same direct send method as used in establishing dynamic processes */
```

Comment:
Can this collide with other dynamic process management messages?

Reply:
If we some how have accumulated `0xF000` dynamic processes ... so yes, there is a collision possibility, although very unlikely. Ideally, we should reserve the space just for finalize.